src/content/state-taxes/louisiana.md

```diff
@@ -16,7 +16,7 @@ updated: 2023-02-15
 
 ### Travel Card
 
-* [R-1376 Government Employees Hotel Lodging Sales/Use Tax Exemption Certificate](https://revenue.louisiana.gov/Forms/ForBusinesses)
+* [R-1376 Government Employees Hotel Lodging Sales/Use Tax Exemption Certificate [PDF, 2 pages]](/files/R_1376_Louisiana_State_Tax_Travel_Form.pdf)
```

Comment:
This should be removed from this PR, This change is not desired.  See PR 695.

Reply:
Updated sorry started from a branch that wasnt staging